server/src/test/java/io/crate/planner/optimizer/OptimizerRuleSessionSettingProviderTest.java

```diff
@@ -62,22 +61,26 @@ public void test_optimizer_rule_session_settings() {
         assertThat(sessionSetting.description(), is("Indicates if the optimizer rule MergeFilters is activated."));
         assertThat(sessionSetting.defaultValue(), is("true"));
 
-        var mergefilterSettings = new SessionSettings("user",
-                                                      SearchPath.createSearchPathFrom("dummySchema"),
-                                                      true,
-                                                      Set.of(MergeFilters.class),
-                                                      true);
+        SearchPath searchPath = SearchPath.createSearchPathFrom("dummySchema");
```

Comment:
nit: why not inline it?

Reply:
I had used it further below before I made some more changes that made that unnecessary and didn't bother changing it back.

Local variables don't really cost anything and it is a test.